CMakeLists.txt

```diff
@@ -1,16 +1,12 @@
-cmake_minimum_required(VERSION 3.5)
+cmake_minimum_required(VERSION 3.13)
 
 #-----------------------------------------------------------------------------
 # Enable C++11
 #-----------------------------------------------------------------------------
 set(_msg "Setting C++ standard")
 message(STATUS "${_msg}")
 if(NOT DEFINED CMAKE_CXX_STANDARD)
-  if("${Slicer_VTK_VERSION_MAJOR}" STREQUAL "8" OR DEFINED Qt5_DIR)
```

Comment:
Let's keep this in for now and apply it when effectively removing support for Qt4/VTK7.